internal/operators/lvm/config.go

```diff
@@ -0,0 +1,32 @@
+package lvm
+
+import (
+	"fmt"
+
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+)
+
+type Config struct {
+	LvmCPUPerHost          int64  `envconfig:"LVM_CPU_PER_HOST" default:"1"`
+	LvmMemoryPerHostMiB    int64  `envconfig:"LVM_MEMORY_PER_HOST_MIB" default:"1200"`
+	LvmMinDiskSizeGB       int64  `envconfig:"LVM_MIN_DISK_SIZE_GB" default:"0"`
+	LvmMinOpenshiftVersion string `envconfig:"LVM_MIN_OPENSHIFT_VERSION" default:"4.11.0"`
+}
+
+// count all disks of drive type ssd or hdd
+func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string) (int64, error) {
+	var countDisks int64
+	var err error
+
+	for _, disk := range disks {
+		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
+			if disk.SizeBytes < conversions.GbToBytes(o.config.LvmMinDiskSizeGB) {
+				err = fmt.Errorf("ODF LVM requires all the non-bootable disks to be more than %d GB", o.config.LvmMinDiskSizeGB)
```

Comment:
Why return an error for a disk that won't be included?  Just don't include it.  With the current code if I have one disk less than LvmMinDiskSizeGB then ValidateHost will return an error.

Reply:
(There's no need to return error at all - just return countDisks - if it's 0 then the caller knows that there are no valid disks)